### PR TITLE
Bumping elixir to 1.9 and erlang to 22

### PR DESCRIPTION
**Pull request by hez**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 language: elixir
-elixir: '1.7.3'
-otp_release: '21.3'
+elixir: '1.9.2'
+otp_release: '22.0.7'
```
Comment (stewart) [inline .travis.yml]: This and `.tool-versions` should likely be synchronized to use the same Erlang/OTP version.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 - Refactor caching servers startup, no longer started by ShopifyAPI.Application
 - Fix nested tuple returns on things like Shopify timeouts.
   WARNING This is a breaking change, errors returned as `{:error, {:error, any()}}` will be `{:error, any()}` now.
+- Bumped Elixir language ver to 1.9
```
Comment (stewart) [inline CHANGELOG.md]:
```suggestion
- Increased minimum supported version of Elixir to `v1.9.0`.
```

File: .travis.yml (hunk 1 above)
Comment (hez) [reply .travis.yml]: good catch, not sure how tool ver got bumped

File: .travis.yml [hunk 3]
```diff
@@ -11,7 +11,7 @@ env:
 
 before_script:
   - mix do deps.get, compile --warnings-as-errors, coveralls.json
-  - MIX_ENV=dev travis_wait 20 mix dialyzer --plt
+  - MIX_ENV=dev travis_wait 30 mix dialyzer --plt
```
Comment (stewart) [inline .travis.yml]: 😬

Comment (hez) [reply .travis.yml]: travis has been timing out running dialyzer, as the plt cache is invalidated with the elixir ver bump

**Pull request merged**